packages/web/src/features/activity-certificate/frames/ActivityCertificateInfoFrame/ActivityCertificateInfoSecondFrame.tsx

```diff
@@ -0,0 +1,372 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import Card from "@sparcs-clubs/web/common/components/Card";
+import Info from "@sparcs-clubs/web/common/components/Info";
+
+import Icon from "@sparcs-clubs/web/common/components/Icon";
+import TextInput from "@sparcs-clubs/web/common/components/Forms/TextInput";
+import Button from "@sparcs-clubs/web/common/components/Button";
+import DateRangeInput from "@sparcs-clubs/web/common/components/Forms/DateRangeInput";
+import { ActivityCertificateFrameProps } from "../ActivityCertificateNoticeFrame";
+// eslint-disable-next-line no-restricted-imports
+import { ActivityDescription } from "../../types/activityCertificate";
+
+const StyledCard = styled(Card)<{ type: string }>`
+  padding: 32px;
+  gap: 20px;
+  align-self: stretch;
+`;
+
+const ActivityCertificateSecondFrameInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 20px;
+  align-self: stretch;
+`;
+
+const ActivityCertificateRow = styled.div`
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 8px;
+  display: inline-flex;
+  flex-direction: row;
+`;
+
+const IconOuterFrameInner = styled.div`
+  height: 36px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  justify-content: space-between;
+  align-items: flex-start;
+  display: flex;
+`;
+
+const IconInnerFrameInner = styled.div`
+  padding: 8px;
+  border-radius: 4px;
+  overflow: hidden;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+  display: flex;
+  cursor: pointer;
+`;
+
+const InputFrameInner = styled.div`
+  flex: 1 1 0;
+  /* height: 36px; */
+  justify-content: center;
+  align-items: flex-start;
+  gap: 20px;
+  display: flex;
+`;
+
+const DescriptionInputFrameinner = styled.div`
+  width: 100%;
+  /* height: 36px; */
+  justify-content: flex-start;
+  align-items: center;
```

Comment:
그리고 이거 바꾸면 layout 맞을거에용
```suggestion
  align-items: flex-start;
```

Reply:
오오 위 4개 코멘트에서 알려준 방법대로 하면 높이 문제는 훨씬 간단하게 해결되는구만! 지식이 늘었습니당
근데 저렇게 되면 아래 사진처럼 빨간 줄이 지워지고 밑에 에러 글이 적히는 공간이 늘어나지 않고 빨간 줄을 띄울 방법이 없어서 (혹은 더 복잡해서) 그냥 지금 있는대로 처리해본거였어염... 혹시 더 나은 방법이 없으면 원래대로 두려고 하는데 의견 부탁드립니당 ㅠ
<img width="584" alt="스크린샷 2024-05-08 오후 10 18 48" src="https://github.com/academic-relations/ar-002-clubs/assets/43981717/14852501-0f7a-4560-81dd-49c254ba425b">